src/filters/filter-component.js

```diff
@@ -0,0 +1,50 @@
+angular.module('patternfly.filters').component('pfFilter', {
+  bindings: {
+    config: '='
+  },
+  templateUrl: 'filters/filter.html',
+  controller: function () {
+    'use strict';
+
+    var ctrl = this;
+
+    ctrl.$onInit = function () {
+
+      angular.extend(ctrl,
+        {
+          addFilter: addFilter
+        }
+      );
+    };
```

Comment:
I like the use of angular.extend to add functions to ctrl, I'm going to use that in my charts